features/base/participants/constants.js

```diff
@@ -0,0 +1,2 @@
+export const MODERATOR = 'moderator';
```

Comment:
I'd better do this with enum, e.g. like here - https://github.com/jitsi/jitsi-meet-react/pull/36/commits/fb055fc6c1a1c6bf5db02f19cd9d0a8ae3dfdbc5#diff-c3559b1a7139882ee3c1055521f43d68R6

```javascript
/**
 * @enum {string}
 */
const PARTICIPANT_ROLE = {
    PARTICIPANT: 'participant',
    MODERATOR: 'moderator'
};
```
Exposing this as enum then gives opportunity to use values in jsdoc, e.g.
```javascript
```javascript
/**
 * @param {USER_ROLE} role
 */